src/model/User.ts

```diff
@@ -39,29 +40,43 @@ const UserSchema: Schema<User> = new Schema({
         type: String,
         required: [true, "Email is required"],
         unique: true,
-        match: [/^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/, "Please use a valid email"]
+        match: [/^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/, "Invalid email format"]
     },
     password: {
         type: String,
         required: [true, "Password is required"]
     },
     verifyCode: {
         type: String,
-        required: [true, "Verify code is required"]
+        required: [true, "Verification code is required"]
     },
     verifyCodeExpiry: {
         type: Date,
-        required: [true, "Verify code expiry is required"]
+        required: [true, "Verification code expiry is required"]
     },
     isVerified: {
         type: Boolean,
         default: false
     },
-    isAcceptingMessages: { // Consistent naming
+    isAcceptingMessages: {
         type: Boolean,
         default: true
     },
-    messages: [MessageSchema]
+    messages: [{ type: Schema.Types.ObjectId, ref: 'Message' }]
```

Comment:
Again messages array has its own type don't need to change it to ObjectId

Reply:
done